weldx/transformations.py

```diff
@@ -985,8 +985,6 @@ def __eq__(self: "CoordinateSystemManager", other: "CoordinateSystemManager"):
 
         if len(graph_0.nodes) != len(graph_1.nodes):
             return False
-        if len(graph_0.edges) != len(graph_1.edges):
-            return False
```

Comment:
Removed it, since the number of edges is currently bound to the number of nodes (factor 2) - no chance to get this branch triggered